utils/songtreasures.ts

```diff
@@ -0,0 +1,10 @@
+import type { TrackModel } from "@bcc-code/bmm-sdk-fetch";
+
+export default function getSongtreasuresLink(track: TrackModel) {
+  if (!trackIsSong(track)) return undefined;
+
+  const song = track.songbookRelations?.[0];
+  if (!song) return undefined;
+
+  return `https://songtreasures.app/songs/${songbookName(song.name)}/${song.id}`;
```

Comment:
add comment:
// This always opens the norwegian songbook, since SongTreasures uses the name of the songbook to determine the language. HV = norwegian, WOTL = english, WDH = german